pymeasure/instruments/chroma/chroma_63600.py

```
@@ -0,0 +1,492 @@
+            CCFSL|CCFSM|CCFSH
+            TIML|TIMM|TIMH
+            SWDL|SWDM|SWDH
+            OCPL|OCPM|OCPH
+            PROG
+            MPPTL|MPPTM|MPPTH
+            UDWL|UDWM|UDWH
+        """,
+        validator=strict_discrete_set,
+        values=MODES,
+        get_process=lambda v: (InstrMode(v),) if v=="PROG" else \
+                            (InstrMode(v[:-1]),InstrModeRange(v[-1])),
```

Flake8 (7.3.0)

[error] continuation line unaligned for hanging indent

(E131)